python/src/LHSExperiment_doc.i.in

```diff
@@ -13,6 +13,16 @@ distribution : :class:`~openturns.Distribution`
     set of input data.
 size : positive int
     Number :math:`\mathrm{card}\,I` of points that will be generated in the experiment.
+initialize : bool
+    Flag to tell if the shuffle must be regenerated at each call to generate
+    or not.
+    Default is *False*: the shuffle is generated once and for all.
```

Comment:
I find this member name misleading (alwaysShuffle?); moreover the same member name is added to LowDiscrepancyExperiment, but with a different default value, this is very confusing.